### PR TITLE
Remove the `assigner` prop of issue event

### LINKED ISSUE #20795: Issue event `assigned` does not have `assigner`
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#assigned

https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#unassigned

### What part(s) of the article would you like to see updated?

https://github.com/github/docs/blob/fed0a6c56ee6c6cce63ff06ac760edc5439aa7d7/data/reusables/issue-events/assignee-properties.md?plain=1#L1-L2

As described in the `github/rest-api-description` repo, the timeline api returns the `assigned` event as follows:

```yaml
timeline-assigned-issue-event:
  title: Timeline Assigned Issue Event
  description: Timeline Assigned Issue Event
  type: object
  properties:
    id:
      type: integer
    node_id:
      type: string
    url:
      type: string
    actor:
      "$ref": "#/components/schemas/simple-user"
    event:
      type: string
    commit_id:
      type:
      - string
      - 'null'
    commit_url:
      type:
      - string
      - 'null'
    created_at:
      type: string
    performed_via_github_app:
      anyOf:
      - type: 'null'
      - "$ref": "#/components/schemas/integration"
    assignee:
      "$ref": "#/components/schemas/simple-user"
```

the field `assigner` is not included.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yanyongyu**
### Why:

Closes #20795 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Remove the `assigner` property of the timeline `assigned` and `unassigned` event. The `assigner` property is not included in `github/rest-api-description` and `octokit/webhooks` any more.

<details>
<summary>event schema</summary>

```yaml
timeline-assigned-issue-event:
  title: Timeline Assigned Issue Event
  description: Timeline Assigned Issue Event
  type: object
  properties:
    id:
      type: integer
    node_id:
      type: string
    url:
      type: string
    actor:
      "$ref": "#/components/schemas/simple-user"
    event:
      type: string
    commit_id:
      type:
      - string
      - 'null'
    commit_url:
      type:
      - string
      - 'null'
    created_at:
      type: string
    performed_via_github_app:
      anyOf:
      - type: 'null'
      - "$ref": "#/components/schemas/integration"
    assignee:
      "$ref": "#/components/schemas/simple-user"
```

</details>

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
@yanyongyu Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by PIYUSH-MISHRA-00 — COMMENTED** [review]
Yeah fine it should work...

File: data/reusables/issue-events/assignee-properties.md [hunk 1]
```diff
@@ -1,2 +1 @@
 `assignee` | `object` | The person unassigned from this issue.
```
Comment (skedwards88) [inline data/reusables/issue-events/assignee-properties.md:1]: Thanks for realizing that this field exists for the Issue events API but not the Timeline events API! With that in mind, maybe we should keep this line but add that it is only available for one of the APIs. How does this sound?

```suggestion
`assignee` | `object` | The person unassigned from this issue.
`assigner` | `object` | The person who performed the unassignment for this issue. This field is available in the Issue events API but not the Timeline events API.
```

**Review by skedwards88 — APPROVED** [review]
🚀 Thanks again! We'll get this merged down for you.

**Pull request merged**